analyzer/restapi-plugin/src/main/resources/application.properties

```diff
@@ -4,15 +4,13 @@ server.tomcat.connection-timeout=60000
 server.port=8080
 server.tomcat.threads.max=20
 kb.user=${db.user}
-kb.java.url=${db.java.url}
-kb.python.url=${db.python.url}
-kb.c.url=${db.c.url}
-kb.maven.depgraph.path=${dg.maven.path}
-kb.pypi.depgraph.path=${dg.pypi.path}
-kb.debian.depgraph.path=${dg.debian.path}
+kb.url=${db.url}
+kb.depgraph.path=${dg.path}
 kb.graphdb.path=${gd.path}
 kb.vulnchains.path=${vc.path}
 lima.rcg.url=${rcg.url}
+dependency.resolver.address=${dep.api}
```

Comment:
The code seems to automatically prepend http:// to this value, which I don't think is a good idea.
In deployment you should be able to write: `--dep.api=http://fasten-pypi-resolver:5000` but now you can only do `--dep.api=fasten-pypi-resolver:5000`.

Reply:
Agreed. Addressed on this commit: 49c9c91